wildfly/server-subsystem/src/main/config/default-server-subsys-config.properties

```diff
@@ -67,4 +67,7 @@ keycloak.server.subsys.default.config=\
               </properties>\
         </provider>\
     </spi>\
+    <spi name="login-protocol">\
+        <provider name="docker-v2" enabled="false"/>\
+    </spi>\
```

Comment:
Profiles should be used to control activation of this feature. See [here](http://www.keycloak.org/docs/3.0/server_installation/topics/profiles.html).

@stianst I guess we have a policy in place that all features are enabled by default, but users can disable them?